CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
+## [10.1.0] - 2019-12-04
```

Comment:
I think Lewis might have a PR in with this same version number

Reply:
I'm not particularly attached to it in away way... feel free to grab it